src/commands/deployment/README.md

```diff
@@ -27,6 +27,39 @@ datadog-ci deployment mark --env prod --service payment-service --revision v1.1.
 - `--tags` is an array of key value pairs in the format `key:value`. These tags are added to the deployment event shown in Datadog.
 - `--no-fail` (default: `false`) will prevent the deployment command from failing if there are issues submitting the data.
 
+### `correlate`
+
+The `correlate` command "connects" the CD deployments with a CI pipeline. If you are using `datadog-ci deployment mark` then you do not need to use 
+this command and the correlation will happen automatically. This command does not work for every setup, check the section 
+[when to call the command][#When-to-call-the-command] for more details.
+
+This command will allow Datadog to correlate the associated deployments to a pipeline from the UI. It also works the other way around, 
+so that given a deployment you can check the pipeline that triggered it.
+
+#### When to call the command
+
+For the command to work properly, you need to be making changes to the configuration repository (where the kubernetes manifests are) from
+the CI using git. More specifically the  command needs to be called **after committing the changes and before pushing them**:
+
+1. Make the changes to the configuration (for example: updating a image tag)
+2. `git commit -m "updating my kubernetes configuration"`
+3. `datadog-ci deployment correlate` (you can check below the command syntax)
```

Comment:
I'd at least add `--provider` if it's mandatory. You can say "see below for more options"